src/styles/pages/_pagination.scss

```diff
@@ -0,0 +1,46 @@
+@import '../settings/colors';
+
+$W_PAGINATION_LINK_SPACING: 6px;
+
+@mixin w-pagination__links($COLOR, $ACTIVE: false) {
+  display: flex;
+  font-size: 20px;
+  justify-content: center;
+  min-width: 20px;
+  padding: 0px $W_PAGINATION_LINK_SPACING;
```

Comment:
I think we may need to increase this padding (perhaps adding some to the top and bottom as well) such that the tap target size is 40 x 40. I think right now it's around 23 x 32.